### ISSUE TITLE
Licensing

### ISSUE THREAD

**Issue by philkunz** [alexmacarthur/typeit#226]
@alexmacarthur 

I'm currently looking at this lib for a project, and I noticed a few licensing things.

This lib is developed under GPL 2. Any contributors also submit their code under this license.
Anyone distributing a software project that includes this lib as dependency has to make his/her sourcecode available upon request.

At the same time there is the offer of an commercial license. The GPL 2 allows the use of the software in commercial projects by default. There is no need for any extra license for commercial projects besides GPL 2.

So...... Is the mentioned extra commercial license free of GPL 2?
If yes: Has the issuer of the extra commercial license the right to issue a GPL free license? Because contributors have contributed under GPL 2... I kind of doubt that.

In other words:

1. is this lib safe for commercial use?
2. is this lib burnt for closed source code?
3. does the issuer of the extra commercial license issue invalid GPL 2 free licenses?
4. Or what?

**Comment by philkunz** [comment]
So after a quick consultation with our legal counsel, this seems to be a legit problem... We settled on the MIT licensed typed.js for now.

**Comment by alexmacarthur** [comment]
Hi, @philkunz -- you bring up some good questions that, honestly, largely have to do with my own oversight. Let me think about how I want to move forward with licensing in this project, and I'll report back on this thread. 

Thanks!
Alex

**Comment by philkunz** [comment]
How was this addressed?
